docs/dev/install.rst

```diff
@@ -175,7 +175,9 @@ you can use ``celery.contrib.rdb``:
 
 .. code-block:: python
 
-    from celery.contrib import rdb; rdb.set_trace()
+    from celery.contrib import rdb
+
+    rdb.set_trace()
```

Comment:
This feels like somewhere that we want the existing syntax, as it's standard for using PDB-like things. Is there a way to disable the auto-formatting here?

Reply:
It's possible to use the new example, arguing like this:

1. Example code is blackened, so someone copy-pasting won't have to bother with linting errors from copy-pasting it.
2. Importing `rdb` in itself is harmless, but if we only comment out `set_trace`, we'll get an "unused import" linting error anyways
3. We can start the trace at any point in our code, the example of using the module for instance doesn't have the line above... https://github.com/celery/celery/blob/master/celery/contrib/rdb.py

Reply:
Also, `blacken-docs` is just a wrapper around `black`, so we can use all the features of `black`, like disabling auto-formatting.
I tried this example and it worked:
```
.. code-block:: python

    # fmt: off
    from celery.contrib import rdb; rdb.set_trace()
```

Let me know if we need to update the PR.

cc: @benjaoming @ericholscher

Reply:
Also, `blacken-docs` is just a wrapper around `black`, so we can use all the features of `black`, like disabling auto-formatting.
I tried this example and it worked:
```
.. code-block:: python

    # fmt: off
    from celery.contrib import rdb; rdb.set_trace()
```

Let me know if we need to update the PR.

cc: @benjaoming @ericholscher

Reply:
Great work @dojutsu-user - glad to see that the possibility exists.

I think that it's great to have documentation that lints with black and a little odd/distracting to have comments in documentation examples for disabling linting rules. My concern would be that the reader is confused.

What do you think, @ericholscher ?

Reply:
Hrm, I'd really like to avoid having the comment in the code block, as that will confuse users as well.

This code should never be checked into a repo, since it will block and break if it's ever checked in. I'm not really sure what you're trying to say @benjaoming?

I guess we can just move forward with this change, but I do think it's overall worse in the docs because of the lack of flexibility here.

Reply:
The new linting caught a wrong indentation and made it easier to copy-paste without re-formatting, so I think that the benefits can definitely outweigh the disadvantages :+1: 

A possible approach is to say that when/if there's a code example that does not conform to black, then there must be a better alternative with black formatting.

> This code should never be checked into a repo, since it will block and break if it's ever checked in. I'm not really sure what you're trying to say @benjaoming?

Well, you could deploy it to non-prod in order to debug a development environment. Yes, you are right that it would be great and flexible if the linting could be disabled for a segment that intentionally doesn't need it. In this case, it seems possible to accept, unless there's a way to disable it without adding comments in the code itself.